src/ir/info.rs

```diff
@@ -22,6 +22,7 @@ pub enum Info {
         name: ast::Id,
         bind_loc: GPosIdx,
         delay_loc: GPosIdx,
+        interface_port: Option<super::InfoIdx>,
```

Comment:
Why does this track an `InfoIdx` instead of storing the information inline?

Reply:
I can change this, but then the fields would be unnamed which might be not as good?

Would we rather have two fields `interface_port_name` and `interface_port_bind_loc` instead, or add a whole new struct?

Reply:
Either is fine I think. I've just been adding as many field to the variants because the `Info` object size doesn't matter too much